packages/core/src/components/Row/index.tsx

```diff
@@ -50,6 +52,7 @@ export class Row extends React.Component<Props & WithStylesProps> {
       compact,
       inline,
       maxHeight,
+      minHeight,
```

Comment:
mind adding a story to show this new prop too?

Reply:
@stefhatcher It seemed like the paradigm was not to add stories for these types of props.  There was no maxHeight story either

Reply:
Oh I was looking at row.  Sure will add for both